indico/modules/events/editing/client/js/editing/timeline/selectors.js

```diff
@@ -38,18 +38,20 @@ export function processRevisions(revisions) {
   const newRevisions = [];
   let numberOfRevisions = 1;
   let currentRevision = revisions[0];
+  let previousRevision = null;
   let items = [...currentRevision.comments];
 
   for (const [index, revision] of revisions.entries()) {
     const {initialState, finalState} = revision;
-
     if (!currentRevision) {
       currentRevision = {...revision};
       items = [...currentRevision.comments];
     }
 
     if (initialState.name === InitialRevisionState.needs_submitter_confirmation) {
-      currentRevision.header = Translate.string('Editor has made some changes to the paper');
+      currentRevision.header = `${previousRevision.editor.fullName} ${Translate.string(
+        '(editor) has made some changes to the paper'
+      )}`;
```

Comment:
Use a placeholder in the translatable string - the name may be somewhere else in other languages.